dist/comment_style.py

```diff
@@ -131,7 +132,8 @@ def validate(line, expect_match):
         -o \( -name \"*.in\" ! -name \"Makefile.in\" \) \
         -o -name \"*.cpp\" -o -name \"*.i\" "
     if args.fast:
-        command = "git diff --name-only $(git merge-base --fork-point develop) bench \
+        since_dev = common_functions.last_commit_from_dev
```

Comment:
Missing `()` at the end of `last_commit_from_dev`?

Reply:
Good catch! This was printing the function pointer but the `subprocess.run` was returning an empty string as if there was no diff.